src/commands/autocomplete/values.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import {flags} from 'cli-engine-command'
+import HerokuCommand from 'cli-engine-heorku'
+import Plugins from '../../plugins'
+import ACCache from '../../cache'
+import path from 'path'
+
+export default class AutocompleteValues extends HerokuCommand {
+  static topic = 'autocomplete'
+  static command = 'values'
+  static description = 'generates autocomplete values'
+  static hidden = true
+  static flags = {
+    // don't require cmd or flag
+    // we want it to fail silently
+    // or autocomplete gets weird
```

Comment:
missing flag errors will throw and that will show up as the autocomplete options. there maybe be ways around this (sending stderr to null?) but i haven't explored those yet.